lua/lazyvim/plugins/extras/lang/tex.lua

```diff
@@ -0,0 +1,61 @@
+return {
+  {
+    "folke/which-key.nvim",
+    opts = {
+      defaults = {
+        ["<LocalLeader>l"] = { name = "+vimtex" },
+      },
+    },
+  },
+
+  -- Add BibTeX/LaTeX to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "bibtex", "latex" })
+      end
+      if type(opts.highlight.disable) == "table" then
+        vim.list_extend(opts.highlight.disable, { "latex" })
+      else
+        opts.highlight.disable = { "latex" }
+      end
+    end,
+  },
+
+  {
+    "lervag/vimtex",
+    version = "*",
+    ft = { "bib", "tex" },
+    keys = {
+      { "<LocalLeader>ld", "<Cmd>VimtexDocPackage<CR>", desc = "Open Documentation" },
+    },
+    config = function()
+      vim.opt.conceallevel = 2
+      vim.g.vimtex_complete_enabled = 0 -- use texlab for completion
+      vim.g.vimtex_mappings_disable = { ["n"] = { "K" } } -- disable `K` as the confilict to LSP hover
```

Comment:
Again, LSP hover and VimTeX `keywordprg` are different and complementary.

Reply:
We should probably have both available.

I'm fine with the texlab lsp hover as `K` by default, unless the opposite makes more sense.

You can add the other mapping to:

```lua
nvim-slpconfig.opts.servers.texlab.keys = {
  {"<c-k>", "<Plug>(vimtex-doc-package)", desc = "Vimtex Docs" }
}
```

Reply:
Sorry, `<c-k>` is not a keymap we should use since that's to go to the window above :)
Any suggestions? `<leader>k`?

Reply:
I think I had already mapped this function to `<localleader>ld`, where `<localleader>l` is VimTeX's default "leader" key and `<localleader>lk` is occupied. I removed this keymap temporarily in the lazy-loading related commit because the keybinding will always be available without lazy-loading.